LEGO1/lego/legoomni/src/entity/legocarraceactor.cpp

```diff
@@ -27,20 +31,68 @@ LegoCarRaceActor::LegoCarRaceActor()
 	VariableTable()->SetVariable(g_fuel, "0.8");
 }
 
-// STUB: LEGO1 0x10080590
-void LegoCarRaceActor::FUN_10080590(float)
+// FUNCTION: LEGO1 0x10080590
+// FUNCTION: BETA10 0x100cd8cf
+void LegoCarRaceActor::FUN_10080590(float p_float)
 {
+	MxFloat maxSpeed = m_maxLinearVel;
+	Mx3DPointFloat destEdgeUnknownVector = Mx3DPointFloat();
```

Comment:
This should be written as `Mx3DPointFloat destEdgeUnknownVector;` - it's functionally equivalent but more conventional.